src/com/nisovin/magicspells/castmodifiers/conditions/VarStringEqualsCondition.java

```diff
@@ -0,0 +1,67 @@
+package com.nisovin.magicspells.castmodifiers.conditions;
+
+import org.bukkit.Location;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+
+import com.nisovin.magicspells.DebugHandler;
+import com.nisovin.magicspells.MagicSpells;
+import com.nisovin.magicspells.castmodifiers.Condition;
+
+public class VarStringEqualsCondition extends Condition {
+
+	String variable;
+	String value;
+	
+	@Override
+	public boolean setVar(String var) {
+		try {
+			String s = var;
+			int colonposition = var.indexOf(":");
+			
+			//Checks if there is a colon to begin with.
+			if (colonposition == -1) {
+				MagicSpells.error("Improper varstring comparison. Use : to compare the string of a variable.");
+				throw new Exception("Invalid modifier format!");
```

Comment:
I like the idea of being more verbose in the problem, although you may want to throw a more detailed exception type.